core/src/main/kotlin/org/jetbrains/kotlinx/dataframe/impl/api/convertTo.kt

```diff
@@ -280,10 +275,16 @@ internal fun AnyFrame.convertToImpl(
     var result = convertToSchema(marker.schema, emptyPath())
 
     dsl.fillers.forEach { filler ->
-        val paths = result.getColumnPaths(filler.columns)
-        missingPaths.removeAll(paths.toSet())
-        result = result.update { paths.toColumnSet() }.with {
-            filler.expr(this, this)
+        val paths = result.getColumnPaths(UnresolvedColumnsPolicy.Create, filler.columns).toSet()
```

Comment:
could you please add a comment with the high-level idea of this algorithm starting from dsl.fillers